### PR TITLE
Bugfix/boolean crash

### PR DESCRIPTION
**Pull request by tvand7093**
I believe this should resolve the issues mentioned in #21. However, I have a few questions.
1. Is there a more succinct way to do the true/false checks I added? 
2. I used 1 to denote a true value since this is currently iOS (following the values for YES/NO). Is this good or is there some other ruby format for this kind of thing?

### CODE REVIEW

File: lib/ProMotion/form/form.rb [hunk 1]
```diff
@@ -71,6 +71,12 @@ def input_data(input)
       data[:title] ||= input[:label] || input[:name ].to_s
       data[:cell ] ||= input[:cell_class] if input[:cell_class]
 
+      # valid boolean checks
+      if input.key?(:value) && input[:type] == :boolean
```
Comment (markrickert) [inline lib/ProMotion/form/form.rb:78]: His line will fail when value is false.

Comment (markrickert) [inline lib/ProMotion/form/form.rb:78]: Perhaps check !nil?

Comment (tvand7093) [inline lib/ProMotion/form/form.rb:78]: I was under the impression that `input.key?(:value)` merely checked if the key existed, and not actually look at the value. Is this not the case? PS: not a Ruby pro so please ignore my ignorance :wink:

Comment (jamonholmgren) [inline lib/ProMotion/form/form.rb:78]: Yeah, actually you're right @tvand7093 .

**Comment by jamonholmgren** [comment]
@tvand7093 Can you bring this up to date with master? I'll merge it, if so.

**Pull request closed without merging**